charts/forseti-security/values.yaml

```diff
@@ -18,6 +18,9 @@ server:
   # server.cloudsqlConnection is the connection to the cloudsql instance
   cloudsqlConnection: ""
 
+  # cloudsqlDatabaseName is the name of the Forseti Cloud SQL database
+  cloudsqlDatabaseName: forseti_security
+
```

Comment:
Can we consider moving this to a separate section?  I think as we add more variables (e.g. username and password) a separate section will be more clear. For example:
```yaml
server:
...
orchestrator:
...
database:
  name: forseti_security
```
I know this will require changes to the related Terraform PR. If you don't have cycles to do this at the moment, I can re-approve and we submit as is.

Reply:
Good suggestion! I have updated this PR and [TF #314](https://github.com/forseti-security/terraform-google-forseti/pull/314) to support this. Tested and verified Forseti is able to connect to DB with custom name.